src/types.rs

```diff
@@ -164,6 +164,7 @@ pub struct Response {
 
 #[pymethods]
 impl Response {
```

Comment:
@sansyrox  We need to make change here to check for headers and create the Response accordingly?

Reply:
@ParthS007 , in the future yes. But would we need that for now too? 

I suppose you are suggesting for things like json content?